atomic_reactor/cli/main.py

```diff
@@ -74,7 +74,9 @@ def construct_kwargs(**kwargs):
     recognized_kwargs = ['image', 'parent_registry', 'parent_registry_insecure',
                          'target_registries', 'target_registries_insecure',
                          'dont_pull_base_image']
-    is_recognized_kwarg = lambda x: x in recognized_kwargs or x.startswith('source__')
```

Comment:
Out of curiosity, was the linter actually complaining about this?

Reply:
atomic_reactor/plugins/pre_inject_yum_repo.py:23:5: E731 do not assign a lambda expression, use a def